library/modules/bigip_ssl_csr.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2019, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_ssl_csr
+short_description: Create SSL CSR files on the BIG-IP
+description:
+  - This module will create SSL CSR files on a BIG-IP. CSRs
+    require an associated SSL key to prexist on the BIG-IP.
+version_added: "f5_modules 1.2"
+options:
+  name:
+    description:
+      - The name of the csr file.
+    type: str
+    required: True
+  commonName:
```

Comment:
wrong parameter name, no camel case

Reply:
Gah! I wrote all the testing correctly, just made this documentation mistake! Fixed.